.meteor/packages

```diff
@@ -56,3 +55,6 @@ kadira:flow-router
 scorpiusjs:config
 scorpiusjs:materialize-admin-only
 useraccounts:bootstrap
+alethes:pages
+kurounin:pagination
+kurounin:pagination-blaze
```

Comment:
Hi Derek. Just starting a code review now. So, taking a deep breath, heh... Have you given any thought whether there might be any way we could avoid adding these pagination packages? I ask because our app has grown heavy with dependencies, which have to be loaded all together for now (until the next Meteor release changes this). I suppose we could just leave these for now, and put moving them into dynamic imports on the "one day / maybe" list. But anyway, I'm just starting to review this so let me see what these packages have to offer :)